firstvoices/backend/tests/test_apis/base/base_uncontrolled_site_api.py

```diff
@@ -177,6 +177,8 @@ def get_expected_standard_fields(self, instance, site):
             "createdBy": instance.created_by.email,
             "lastModified": instance.last_modified.astimezone().isoformat(),
             "lastModifiedBy": instance.last_modified_by.email,
+            "systemLastModified": instance.system_last_modified.astimezone().isoformat(),
+            "systemLastModifiedBy": instance.system_last_modified_by.email,
             "id": str(instance.id),
```

Comment:
I see these tests that check the serialized fields against the instance, which is great. I think we do need tests that check the date and user against the expected values. E.g., that the system modified user is the request user, and that the system modified date is either equal to or greater than the created date depending whether it is a create or update. If I missed them please just let me know! If not, I'm happy to chat about the easiest way to add those if needed.

Reply:
Added create, update and patch tests to `base_uncontrolled_site_api` and update and patch tests to `test_sites_api` as discussed 👍